docs/docs/porting-an-html-site-to-gatsby.md

```diff
@@ -0,0 +1,543 @@
+        <Link to="/who/sam-surgeon">Sam (Tree Surgeon)</Link>
+      </li>
+      <li>
+        <Link to="/who/marin-leafer">Marin (Leafer)</Link>
+      </li>
+    </ul>
+  </Layout>
+)
+```
+
+The `Who We Are` link in `index.js` should now work! Now use the `<Layout>` component in the `index.js` page file too:
+
+```jsx:title=/gatsby-site/src/pages/index.js
+import React from "react"
+import Layout from "../components/Layout" // highlight-line
+
+export default () => (
+  {/* highlight-start */}
+  <Layout>
+    <h1>Welcome To Taylor's Tidy Trees</h1>
+    <h2>We care about trees of all kinds!</h2>
+  </Layout>
+  {/* highlight-end */}
+)
+```
+
+Have a check that the `Who We Are` link is still working.
+
+### Porting other pages
+
+Now it's time for the work to really pay off! Ella's page is a matter of using your `<Layout>` component again and copying in the main content. Don't forget to change `class` to `className`! A `breadcrumbs` prop can be passed to `<Layout>` to set the dynamic page title. Passing props is similar to an attribute on an HTML element:
+
+```jsx:title=/gatsby-site/src/pages/ella-arborist.js
+import React from "react"
+import Layout from "../components/Layout"
+import { Link } from "gatsby"
+
+export default () => (
+  {/* highlight-start */}
+  <Layout breadcrumbs={["Who We Are", "Ella"}}>
+  {/* highlight-end */}
+    <h1>Ella - Arborist</h1>
+    <h2>Ella is an excellent Arborist. We guarantee it.</h2>
+    <div className="bio-card">
+      <img
+        alt="Comically crude stick person sketch"
+        src="/person.png"
+      />
+      <p>Ella</p>
+    </div>
+  </Layout>
+)
+```
+
+The other 2 `Who We Are` pages for Marin and Sam can now be made with a similar structure. Maybe you are even thinking about another component for the Bio Card!
+
+Once `/services` and the root level HTML files are ported, here is what the finished Gatsby project file structure looks like:
+
+```
+gatsby-site
+├── static
+│   ├── favicon.ico
+│   └── person.png
+├── src
+│   ├── styles
+│   │   ├── normalize.css
+│   │   └── style.css
+│   ├── components
+│   │   └── Layout.js
+│   └── pages
+│       ├── index.js
+│       ├── about.js
+│       ├── contact.js
+│       ├── 404.js
+│       ├── who
+│       │   ├── index.js
+│       │   ├── ellla-arborist.js
+│       │   ├── marin-leafer.js
+│       │   └── sam-surgeon.js
+│       └── services
+│           ├── index.js
+│           ├── growing.js
+│           ├── cleaning.js
+│           └── shrinking.js
+├── node_modules
+├── .gitignore
+├── .prettierignore
+├── .prettierrc
+├── gatsby-config.js
+├── LICENSE
+├── package-lock.json
+├── package.json
+└── README.md
+```
+
+## Building & Deploying
+
+With your new Gatsby application taking shape, it's time to integrate it into your existing HTML website.
+
+### Gatsby build step
+
+You now have a site that mirrors the existing HTML site section. Stop the development server if it's still running; it's time to run the production build! 🎉
```

Comment:
```suggestion
With all of the pages ported over, you now have a site that mirrors the existing HTML site. Stop the development server if it's still running; it's time to run the production build! 🎉
```

In case they missed the note about porting over `/services`, this gives them another nudge to catch up. What do you think?